charts/rstudio-connect/templates/deployment.yaml

```
@@ -204,6 +204,9 @@ spec:
       {{- end }}
       {{- if .Values.pod.sidecar }}
 {{ toYaml .Values.pod.sidecar | indent 6 }}
+      {{- end }}
+      {{- with .Values.pod.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ . }}
       {{- end }}
```

```suggestion
      terminationGracePeriodSeconds: {{  .Values.pod.terminationGracePeriodSeconds }}
```
What do you think about just setting this value directly? We set a default integer value of `120` for `terminationGracePeriodSeconds` in all of our charts, but here we're handling it here as if it might be empty or `None`.

---

This is how this looks in the chronicle chart: https://github.com/rstudio/helm/pull/443/files#diff-95db5aca0ede6323eb40498fe2af5fc708cac66d5603bc813e4f979139d7a88cR66